src/components/Skeleton/Skeleton.scss

```diff
@@ -27,6 +27,10 @@ $block: '.#{variables.$ns}skeleton';
                 var(--g-color-base-generic)
             );
             animation: #{variables.$ns}skeleton 1.2s ease-out infinite;
+
+            @media (prefers-reduced-motion: reduce) {
+                animation: none;
```

Comment:
I think we need to set minimal animation duration instead, to not break some code that depends on it

Reply:
Cannot imagine the code that would depend on an infinite animation